portfolio/src/main/webapp/request.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Models the requests sent to the algorithm for scheduling. This class is
+ * useful when the request information needs to be sent to the scheduling
+ * servlet through a POST request. This class contains all the request
+ * information that can be easily converted into a JSON string.
+ */
+class ScheduleRequest {
+    /**
```

Comment:
Nit: Misaligned beginning to javadoc style format.
Expected:
/**
 *
 */
Is:
    /**
   *
   */

Reply:
Done.